pennylane/beta/tapes/qnode.py

```diff
@@ -447,16 +452,7 @@ def to_torch(self, dtype=None):
             self.dtype = dtype or self.dtype or TorchInterface.dtype
 
             if self.dtype is np.complex128:
-                torch_version = list(map(int, torch.__version__.split(".")[:2]))
-                min_version_for_complex = torch_version[0] >= 1 and torch_version[1] >= 6
-
-                if min_version_for_complex:
-                    self.dtype = torch.complex128
-                else:
-                    raise QuantumFunctionError(
-                        "Version 1.6.0 or above of PyTorch must be installed"
-                        "for complex support, such as returning the state"
-                    )
+                self.dtype = torch.complex128

```

Comment:
Not a biggie, but would we want to support other complex number types?

Reply:
I think it's not needed for this state PR - we are choosing `np.complex128` for the state dtype earlier on and then just converting here.

The other use case might be e.g. if a user passes `np.complex64`, but I don't think the `to_torch()` function is very visible.

As a side note, I hope Torch introduce an analogue of [tf.as_dtype()](https://www.tensorflow.org/api_docs/python/tf/dtypes/as_dtype), which would make this easier.